sdk/websites/Azure.ResourceManager.AppService/src/autorest.md

```diff
@@ -137,38 +125,331 @@ rename-rules:
   SSO: Sso
   URI: Uri
   Etag: ETag|etag
-      
+
+rename-mapping:
+  Site: WebSite
+  AppServiceCertificateOrderPatchResource: AppServiceCertificateOrderPatch
+  AppServiceCertificatePatchResource: AppServiceCertificatePatch
+  AppServiceEnvironmentPatchResource: AppServiceEnvironmentPatchOptions
+  AppserviceGithubToken: AppServiceGithubToken
+  AppServicePlanPatchResource: AppServicePlanPatchOptions
+  Contact: ContactInformation 
+  Login: LoginInformation
+  MSDeploy: MsDeploy
+  MSDeployLog: MsDeployLog
+  MSDeployLogEntry: MsDeployLogEntry
+  Operation: OperationInformation
+  Recommendation: AppServiceRecommendation
+  Resource: AppServiceResource
+  DetectorResponse: AppServiceDetector
+  ApiKVReference: ApiKeyVaultReference
+  Domain: AppServiceDomain
+# rename property
+  ValidateResourceTypes.Site: WebSite
+  CheckNameResourceTypes.Site: WebSite
+  ApiKVReference.properties.identityType: Identity
+  AppServiceCertificateOrder.properties.autoRenew: IsAutoRenew
+  AppServiceCertificateOrder.properties.expirationTime: ExpireOn
+  AppServiceCertificateOrder.properties.lastCertificateIssuanceTime: LastCertificateIssuedOn
+  AppServiceCertificateOrder.properties.nextAutoRenewalTimeStamp: NextAutoRenewTimeStamp
+  Domain.properties.privacy: HasPrivacy
+  Domain.properties.expirationTime: ExpireOn
+  Domain.properties.autoRenew: IsAutoRenew
+  AppServiceEnvironment.properties.ipsslAddressCount: IPSslAddressCount
+  AppServiceEnvironment.properties.suspended: IsSuspended
+  AppServicePlan.properties.elasticScaleEnabled: IsElasticScaleEnabled
+  AppServicePlan.properties.freeOfferExpirationTime: FreeOfferExpiredOn
+  AppServicePlan.properties.hyperV: IsHyperV
```

Comment:
Is or Enable? Just question

Reply:
This bool type is used to judge if Hyper-V container app service plan, perhaps Is is more appropriate